src/style.css

```diff
@@ -0,0 +1,122 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf) format(ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.big-form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: 8px;
+  margin-right: 8px;
+}
+
+.big-search {
+  position: relative;
+  display: flex;
+
+}
+
+.big-search .big-img {
```

Comment:
use one class for both elements (the big and small img, input, form, etc), because you repeat some styles, and then modify only that values that you need